spring-boot-admin-server-ui/src/main/frontend/services/instance.js

```diff
@@ -67,12 +67,24 @@ class Instance {
   }
 
   async fetchMetric(metric, tags) {
-    const params = tags ? {
-      tag: Object.entries(tags)
+    const params = new URLSearchParams();
+    if (tags) {
+      let firstElementDuplicated = false;
+      Object.entries(tags)
         .filter(([, value]) => typeof value !== 'undefined' && value !== null)
-        .map(([name, value]) => `${name}:${value}`)
-        .join(',')
-    } : {};
+        .forEach(([name, value]) => {
+          params.append('tag', `${name}:${value}`);
+
+          if (!firstElementDuplicated) {
+            // workaround for tags that contains comma
```

Comment:
Do we really need to always duplicate the first param? Would be only needed in very rare case of having only 1 param and the param is containing a comma, right?

Reply:
Yes. I just wrote a simple solution. As duplicate does not break anything.
Do you want me to rewrite code with more precice condition?

Reply:
is ok